.github/workflows/snl-ci.yml

```diff
@@ -61,9 +61,9 @@ jobs:
         working-directory: kokkos/build
         run: ctest --output-on-failure --timeout 3600
 
-  HIP_5_6_1:
-    name: SNL_HIP_ROCM_5_6_1
-    runs-on: [snl-kk-env-openblas-0.3.23-hip-5.6.1-latest]
+  HIP_6_2_1_CPP23:
```

Comment:
No other build currently has a _CPPXY suffix.
Any particular reason to use it here?

Reply:
It distinguishes it from the SNL_HIP_ROCM_6_2_1 job running with similar config but with 20 (L109 below), do you have a preferred naming convention to use?

Reply:
ok.  That's fine.